app/assets/stylesheets/frontend/reset/_reset.scss

```diff
@@ -30,3 +30,10 @@ table, caption, tbody, tfoot, thead, tr, th, td {
   font-weight: normal;
 }
 
+strong {
+  font-weight: bold;
+}
```

Comment:
We specifically don't want to give govspeak parts of whitehall the ability to have bold. So can we either add an override to this in the govspeak or not add this here.